CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.3.1
+- Update some dependencies
+
```

Comment:
according to the release docs this needs to be in an isolated commit.

Reply:
Made the update @jurassix 👍